### PR TITLE
use is_authenticated as property not method

### PR DESCRIPTION
**Pull request by nickpell**
remove warnings:

```
RemovedInDjango20Warning: Using user.is_authenticated() and user.is_anonymous() as a method is deprecated. Remove the parentheses to use it as an attribute.
```

### CODE REVIEW

**Comment by nickpell** [comment]
@dannyroberts

**Comment by dannyroberts** [comment]
does this make any sense?

```
======================================================================
ERROR: test_basic (corehq.apps.api.tests.EWSLocationResourceTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/api/tests.py", line 1689, in test_basic
    self.assert_accessible_via_sessions(self.list_endpoint)
  File "/mnt/commcare-hq-ro/corehq/apps/api/tests.py", line 1660, in assert_accessible_via_sessions
    response = self.client.get(api_url)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 503, in get
    **extra)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 304, in get
    return self.generic('GET', path, secure=secure, **r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 380, in generic
    return self.request(**r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 449, in request
    response = self.handler(environ)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 123, in __call__
    response = self.get_response(request)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 230, in get_response
    response = self.handle_uncaught_exception(request, resolver, sys.exc_info())
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 296, in handle_uncaught_exception
    return callback(request, **param_dict)
  File "/mnt/commcare-hq-ro/corehq/apps/hqwebapp/views.py", line 102, in server_error
    request=request,
  File "/vendor/lib/python2.7/site-packages/django/template/backends/django.py", line 95, in render
    return self.template.render(context)
  File "/vendor/lib/python2.7/site-packages/django/template/base.py", line 204, in render
    with context.bind_template(self):
  File "/usr/local/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/vendor/lib/python2.7/site-packages/django/template/context.py", line 260, in bind_template
    updates.update(processor(self.request))
  File "/mnt/commcare-hq-ro/corehq/util/context_processors.py", line 69, in js_api_keys
    if hasattr(request, 'couch_user') and not request.couch_user.analytics_enabled:
AttributeError: 'NoneType' object has no attribute 'analytics_enabled'
```

**Comment by emord** [comment]
might need to merge https://github.com/dimagi/commcare-hq/pull/12561/files

**Comment by nickpell** [comment]
✅ @esoergel

File: corehq/util/context_processors.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def current_url_name(request):
 
 
 def js_api_keys(request):
-    if hasattr(request, 'couch_user') and not request.couch_user.analytics_enabled:
+    if hasattr(request, 'couch_user') and request.couch_user and not request.couch_user.analytics_enabled:
```
Comment (esoergel) [inline corehq/util/context_processors.py]: nit: you could also do:
```python
if getattr(request, 'couch_user', None) and not request.couch_user.analytics_enabled: 
```

**Review by esoergel — APPROVED** [review]

**Pull request merged**